src/lib/app.ts

```diff
@@ -102,7 +102,9 @@ export default async function getApp(
         ),
     );
 
-    app.use(baseUriPath, patMiddleware(config, services));
+    if (!config.flagResolver.isEnabled('personalAccessTokensKillSwitch')) {
```

Comment:
this will not work as intended. Feature flags are dynamic and can change during runtime of the application. This will either load/not load the middle-ware at startup depending on the flags status on Unleash startup time. 

We need to move the check in to the middleware itself to ensure it can change over time.

Reply:
Yes, I considered this but I thought that the way they want this to work would be ok, but there's no harm in having it as a runtime configuration. It was actually like that before but in this case, I believed the customer wanted it off for good. In any case, let's err for the most general use case which should consider enabling this without having to restart :+1:

Reply:
[9d3e900](https://github.com/Unleash/unleash/pull/3450/commits/9d3e9002c0b46622b19f265a063183f074721afb) should address this